WalletWasabi/Wallets/WalletDirectories.cs

```diff
@@ -68,14 +68,13 @@ public IEnumerable<FileInfo> EnumerateWalletFiles(bool includeBackupDir = false)
 			return result.OrderByDescending(t => t.LastAccessTimeUtc);
 		}
 
-		public string GetNextWalletName(string prefix = "Random Wallet")
+		public string GetNextWalletName(string prefix = "Wallet")
 		{
 			int i = 1;
 			var walletNames = EnumerateWalletFiles().Select(x => Path.GetFileNameWithoutExtension(x.Name));
 			while (true)
 			{
-				var walletName = i == 1 ? prefix : $"{prefix} {i}";
-
+				var walletName = $"{prefix} {i}";
```

Comment:
IMO we shouldn't put the number after name when there is only 1 wallet. `Wallet` is more friendly than `Wallet 1`.
I don't know that but how common is to have same-named wallets?